apps/linode-marketplace-django/roles/post/templates/motd.j2

```diff
@@ -0,0 +1,7 @@
+*********************************************************
+Akamai Connected Cloud Django Marketplace App
+App URL: https://{{ _domain }}
+Credentials File: /root/.credentials
```

Comment:
switch to /home/{{ username }}/.credentials

Reply:
done